.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+**Task name**: <Name of changed or new pipeline task>
+
+**Description**: <Describe your changes here>
+
+**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>
+
+**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>
+
+**Attached related issue:** (Y/N) <Please add link to related issue here>
+
+**Checklist**:
+- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md) how to do it
+- [ ] Checked that applied changes work as expected
```

Comment:
```suggestion
- [ ] Checked that applied changes work as expected
- [ ] Checked that unit tests passed
```

Reply:
Hm... I think it makes sense to cover it by ci instead of running them manually

Reply:
Resolving it - as per offline discussion - that we need to add this to ci